docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -109,6 +112,29 @@ When a Global Admin invites a user to join an account, the Global Admin selects
   - Has Namespace Admin permissions for each Namespace created by that user
 - **Read-Only:** Can only read information
 
+:::note Default Role
+
+When the account is created, the initial user who logs in is automatically assigned the Account Owner role.
+If your account does not have an Account Owner, please reach out to [Support](https://temporalsupport.zendesk.com/) to assign the appropriate individual to this role.
+
+:::
+
+## Using the Account Owner Role
+
+The Account Owner role (i.e., users with the Account Owner system role) holds the highest level of access in the system.
+This role configures account-level parameters and manages Temporal billing and credit data.
```

Comment:
suggestion: maybe "billing and payment information" if "credit data" is referring to credit card data.

Reply:
Changing.

Reply:
(I assume it refers to purchased credits. Please check with your team. For  now, the changes are applied and will be pushed.)